src/Components/CommentCard.jsx

```diff
@@ -1,14 +1,25 @@
 import React from 'react'
+import { Button } from "@material-tailwind/react";
+
+function CommentCard({ comment, selectedUser, handleDeleteComment }) {
+  const handleDelete = () => {
+    handleDeleteComment(comment.comment_id);
+  };
 
-function CommentCard({ comment }) {
-    
   return (
-    <div className='comment-card'>
-        <p>Author: {comment.author}</p>
-        <p>{comment.body}</p>
-        <p>votes: {comment.votes}</p>
+    <div className="comment-card">
+      <p>Author: {comment.author}</p>
+      <p>{comment.body}</p>
+      <p>votes: {comment.votes}</p>
+      {selectedUser === comment.author ? (
```

Comment:
great way to ensure only the user can see the delete button of their own comments!